prisma/schema.prisma

```diff
@@ -7,94 +7,60 @@ datasource db {
   url      = env("DATABASE_URL")
 }
 
-model app_roles {
-  id         Int          @id
-  role       String       @unique(map: "uk_4yak61rfyjr8l0vvud3fa533i") @db.VarChar(255)
-  user_roles user_roles[]
-}
-
 model app_user {
-  id             Int              @id
-  created_at     DateTime?        @db.Timestamp(6)
-  updated_at     DateTime?        @db.Timestamp(6)
-  display_name   String?          @unique(map: "uk_fkrbhagh65rxywp3eh8a64pt8") @db.VarChar(255)
-  password       String           @db.VarChar(255)
-  username       String           @unique(map: "uk_3k4cplvh82srueuttfkwnylq0") @db.VarChar(255)
-  recipe         recipe[]
-  user_favorites user_favorites[]
-  user_roles     user_roles[]
+  id            String          @id @default(uuid())
+  createdAt     DateTime        @default(now()) @map("created_at") @db.Timestamp
```

Comment:
these @map directives let us use camelCase within the app, while still having the preferred snake_case within the db.